<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="push_showText">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Text</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pdbFile">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>15</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>PDB file</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_loadPDB">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>12</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Load PDB</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_target">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>target residue</string>
   </property>
   <widget class="QSpinBox" name="spinBox_atomID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>54</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pdbAtom">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>residue</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_fragment">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>dye fragment</string>
   </property>
   <widget class="QComboBox" name="comboBox_selectBase">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_selectDye">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_selectPosition">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_dye">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>dye</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_base">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>base</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_position">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>position</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="push_addFragment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add fluorophore</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_reloadPDB">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>reload PDB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_docs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_demo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>demo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
